backend/src/main/java/com/eyetrade/backend/model/resource/ArticleResource.java

```diff
@@ -0,0 +1,24 @@
+package com.eyetrade.backend.model.resource;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.annotation.Resource;
+import java.util.UUID;
+
+@Resource
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ArticleResource {
+    private UUID uuid;
+    private String title;
+    private String content;
+    private String stringDate;
+    private String changeDate;
+    private UUID authId;
```

Comment:
I think that should be replaced with the user's name, surname, and email.

Reply:
Because:
1) If we return id then the client app must send another request to learn the name, surname, email of the author.
2) Client app should not know about user ids.

Reply:
yes, it makes sense